pkg/virt-api/webhooks/validating-webhook/admitters/vmi-update-admitter_test.go

```diff
@@ -595,9 +620,42 @@ var _ = Describe("Validating VMIUpdate Admitter", func() {
 			makeDisks(1),
 			makeDisks(1),
 			makeStatus(0, 0),
-			makeExpected("number of disks (1) does not equal the number of volumes (2)", "")),
+			makeExpected("number of disks and filesystems (1) does not equal the number of volumes (2)", "")),
 	)
 
+	Context("with filesystem devices", func() {
+		BeforeEach(func() {
+			enableFeatureGate(virtconfig.VirtIOFSGate)
+		})
+
+		DescribeTable("Should return proper admission response", func(newVolumes, oldVolumes []v1.Volume, newDisks, oldDisks []v1.Disk, filesystems []v1.Filesystem, volumeStatuses []v1.VolumeStatus, expected *admissionv1.AdmissionResponse) {
```

Comment:
This looks good as well, let me know if you want to extract the function or not and I will review after your response.

Reply:
Extracting the function looks cleaner IMO. You can review again

Reply:
@xpivarc can you take another look?

Reply:
Thanks!